circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  node:
+    version: 6.1.0
```

Comment:
I imagine that by the time this matters for us, CircleCI will support v7 of Node.